package.json

```diff
@@ -107,9 +107,6 @@
   "engines": {
     "node": "^6.x.x || ^7.x.x"
   },
-  "jest": {
-    "notify": true
```

Comment:
Because it was slow somehow, we just need to ensure it's a real bug and then track it on our side.